styles/style.css

```diff
@@ -7,11 +7,27 @@
 html, body {
   margin: 0;
   padding: 0;
-  border: 0; }
+  border: 0;
+  font-size: 14px; }
 
 h1 {
-  font-size: 2em; }
+  font-size: 200%; }
 
+@media screen and (max-width: 768px) {
```

Comment:
Wielokrotne powielanie tych samych sekcji utrudnia pracę z kodem i jego czytanie. Możecie zredukować niektóre elementy np. "@media screen and (max-width: 768px)"  który występuje w pliku 6 razy :)